### PR TITLE
turn on ColumnInc sort in topn operator as default

### PR DESCRIPTION
**Pull request by murphyatwork**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] others

## Problem Summary(Required) ：

Turn on the **ColumnInc** Sort in TopN operator.

### CODE REVIEW

**Comment by mofeiatwork** [comment]
@Mergifyio rebase

**Comment by mofeiatwork** [comment]
@Mergifyio rebase

**Review by stdpain — DISMISSED** [review]

**Comment by mofeiatwork** [comment]
run starrocks_clang-format

**Comment by mofeiatwork** [comment]
run starrocks_clang-format

**Comment by mofeiatwork** [comment]
@Mergifyio rebase

File: be/src/exec/vectorized/sorting/sort_column.cpp [hunk 1]
```diff
@@ -138,11 +138,6 @@ class VerticalColumnSorter final : public ColumnVisitorAdapter<VerticalColumnSor
     size_t get_limited() const { return _pruned_limit; }
 
     Status do_visit(const vectorized::NullableColumn& column) {
-        // Fastpath
-        if (!column.has_null()) {
```
Comment (trueeyu) [inline be/src/exec/vectorized/sorting/sort_column.cpp:142]: why remove the fast path?

**Comment by kangkaisen** [comment]
should pass daily test

File: be/src/exec/vectorized/sorting/sort_column.cpp (hunk 1 above)
Comment (mofeiatwork) [reply be/src/exec/vectorized/sorting/sort_column.cpp:142]: This fastpath is wrong, vertical sort for multiple columns has no fastpath.

**Comment by mofeiatwork** [comment]
> should pass daily test

Passed at job 836.

**Review by kangkaisen — APPROVED** [review]

**Review by Pslydhh — APPROVED** [review]

**Review by stdpain — APPROVED** [review]

**Pull request merged**